src/lib/draw/AttributeMode.svelte

```diff
@@ -83,8 +84,8 @@
 
 {#if $currentMode == thisMode}
   {#if $formOpen}
-    <p>Edit attributes to the left, or click another object</p>
```

Comment:
I ditched "another", which maybe makes things less clear. Could have another variation of the singular noun function that omits the article, but was trying to avoid that :)